docker/support/tensorrt_detector/tensorrt_libyolo.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+SCRIPT_DIR="/usr/local/src/tensorrt_demos"
+
+# Clone tensorrt_demos repo
+git clone --depth 1 https://github.com/NateMeyer/tensorrt_demos.git -b conditional_download
+
+# Build libyolo
+cd ./tensorrt_demos/plugins && make all
```

Comment:
@NateMeyer Does this work in CI? The Makefile tries to read your device's CC

Reply:
https://github.com/blakeblackshear/frigate/actions/runs/5479330030/jobs/9981018663#step:11:703

Reply:
Ah, it still compiles, but defaults to the greatest-compatibility `-arch` compute. That's probably fine. Ok.